src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/auditing/LogInAuditHandler.kt

```diff
@@ -0,0 +1,22 @@
+package uk.gov.justice.digital.hmpps.pecs.jpc.auditing
+
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.security.core.Authentication
+import org.springframework.security.web.authentication.AuthenticationSuccessHandler
+import uk.gov.justice.digital.hmpps.pecs.jpc.service.AuditService
+import javax.servlet.http.HttpServletRequest
+import javax.servlet.http.HttpServletResponse
+
+class LogInAuditHandler : AuthenticationSuccessHandler {
```

Comment:
So we should be able to extend the existing class SavedRequestAwareAuthenticationSuccessHandler as follows:

`class LogInAuditHandler(private val auditService: AuditService) : SavedRequestAwareAuthenticationSuccessHandler() {

  override fun onAuthenticationSuccess(
    request: HttpServletRequest?,
    response: HttpServletResponse?,
    authentication: Authentication?
  ) {
    authentication?.let { auditService.createLogInEvent(it.name) }
    super.onAuthenticationSuccess(request, response, authentication)
  }
}
`